packages/core/upload/admin/src/components/AssetDialog/index.jsx

```diff
@@ -214,7 +214,15 @@ export const AssetContent = ({
         </Modal.Title>
       </Modal.Header>
 
-      <Tabs.Root variant="simple" defaultValue={selectedAssets.length > 0 ? 'selected' : 'browse'}>
+      <Tabs.Root
+        variant="simple"
+        defaultValue={selectedAssets.length > 0 ? 'selected' : 'browse'}
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          overflow: 'hidden',
```

Comment:
can you please wrap with styled component to style this

Reply:
Done. Was it what you had in mind ?